clarity/tests/arkadiko-stake-pool-diko-v1-1_tests.ts

```diff
@@ -124,11 +143,11 @@ async fn(chain: Chain, accounts: Map<string, Account>) {
       types.uint(100000000)
     ], wallet_1.address)
   ]);
-  block.receipts[0].result.expectOk().expectUint(475839436);
+  block.receipts[0].result.expectOk().expectUint(91428982948);
 
   // Check DIKO after unstaking
   call = chain.callReadOnlyFn("arkadiko-token", "get-balance", [types.principal(wallet_1.address)], wallet_1.address);
-  call.result.expectOk().expectUint(150375839436);  
+  call.result.expectOk().expectUint(241328982948);  
```

Comment:
@nieldeckx Q: why did this change? You know?

Reply:
Same as other comment - we need to wait 10 days so we have more rewards when unstaking